lib/rules/no-invalid-role.js

```diff
@@ -116,8 +216,23 @@ module.exports = class NoInvalidRole extends Rule {
           ['presentation', 'none'].includes(roleValue) &&
           ELEMENTS_DISALLOWING_PRESENTATION_OR_NONE_ROLE.includes(node.tag)
         ) {
+          if (ELEMENTS_DISALLOWING_PRESENTATION_OR_NONE_ROLE.includes(node.tag)) {
```

Comment:
This looks like a bad merge conflict resolution. This is part of the existing rule logic that doesn't need to be changed.